src/graph/planner/match/MatchSolver.cpp

```diff
@@ -278,7 +278,7 @@ void MatchSolver::buildProjectColumns(QueryContext* qctx, const Path& path, SubP
   DCHECK(!nodeInfos.empty());
   addNode(nodeInfos.back());
 
-  if (!path.anonymous) {
+  if (path.genPath) {
```

Comment:
Is that necessary?

Reply:
`match (v) unwind (v)-[]->() AS p return p`
pattern in unwind statement has no path variable。but need to generate path struct